.github/workflows/build_release_candidate.yml

```diff
@@ -130,3 +134,43 @@ jobs:
             svn add --force .
             svn status
             svn commit -m "Staging Java artifacts for Apache Beam ${{ github.event.inputs.RELEASE }} RC${{ github.event.inputs.RC }}" --non-interactive --username ${{ github.event.inputs.APACHE_ID }} --password ${{ github.event.inputs.APACHE_PASSWORD }} 
+        
+      
+  stage_docker:
+    if: ${{github.event.inputs.STAGE_DOCKER_ARTIFACTS == 'yes'}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: "v${{ github.event.inputs.RELEASE }}-RC${{ github.event.inputs.RC }}"
+          repository: apache/beam
+      - name: Install Java 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - name: Install Python 3.8
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.8'
+      - run: echo $JAVA_HOME
+      - run: echo "JAVA11_HOME=${JAVA_HOME}" >> "$GITHUB_OUTPUT"
+        id: export-java11
+      - name: Install Java 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+      - name: Remove default github maven configuration
+        # This step is a workaround to avoid a decryption issue of Beam's
+        # net.linguica.gradle.maven.settings plugin and github's provided maven
+        # settings.xml file
+        run: rm ~/.m2/settings.xml
```

Comment:
if we migrate to self hosted runner in the future, will remove setting causing problems?

if not, and the self hosted runner does not have this file by default, probably add `|| true` could avoid failure of this step

Reply:
I added a comment to remove this step if we move to self-hosted and added the `|| true` in case the hosted runner setup ever changes

Reply:
Thanks!